examples/basics/scene/magnify.py

```diff
@@ -25,6 +25,7 @@
 # Make a canvas and partition it into 3 viewboxes.
 #
 canvas = vispy.scene.SceneCanvas(keys='interactive', show=True)
+canvas._send_hover_events = True  # temporary workaround
```

Comment:
Ha. I added `_send_hover_events` specifically to support this example and never set the flag.